client/www/scripts/modules/tracing/tracing.controllers.js

```diff
@@ -552,11 +551,19 @@ Tracing.controller('TracingMainController', [
         $scope.resetTracingCtx();
         $scope.selectedPMHost = host;
         $scope.tracingCtx.currentManagerHost = $scope.selectedPMHost;
+        // establish current process count
```

Comment:
Assuming that changePmHost is descriptive, this means that the current cluster size is derived by counting processes (fragile), and is done once, on first connection to the pm? So, if I do `slc ctl set-size` in the background after the change has occurred, tracing will permanently think the size is something that it is not?

@seanbrookes we should do an in-person review, please.